string object_class
float32 x
float32 y
float32 z
